pytential/symbolic/mappers.py

```diff
@@ -515,8 +513,7 @@ def map_int_g(self, expr):
                 for name, arg_expr in expr.kernel_arguments.items()}
 
         return expr.copy(
-                kernel=expr.kernel,
```

Comment:
Any idea why this was here in the first place?

Reply:
In this particular place, I think it's my fault, but I probably just copied it over from one of the other mappers.